velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -335,11 +363,113 @@ TEST_F(ParquetTpchTest, q6) {
   const auto& duckDBRounded =
       fmt::format("select round(revenue, 2) from ({})", duckDbSql);
 
-  auto task = assertQuery(params, filePath, sourcePlanNodeId, duckDBRounded);
+  auto task =
+      assertQuery(params, {filePath}, {sourcePlanNodeId}, duckDBRounded);
 
   const auto& stats = task->taskStats();
   // There should be two pipelines
   ASSERT_EQ(2, stats.pipelineStats.size());
   // We used the default of 10 splits
   ASSERT_EQ(10, stats.numFinishedSplits);
 }
+
+TEST_F(ParquetTpchTest, q18) {
+  const auto& lineitemFilePath = parquetFilesPaths_["lineitem"];
+  const auto& ordersFilePath = parquetFilesPaths_["orders"];
+  const auto& customerFilePath = parquetFilesPaths_["customer"];
+
+  TableColumnsMap tableCols;
```

Comment:
This map is not needed. It would be clearer to use std::vector<std::string> for column names, e.g.

```
std::vector<std::string> lineitemColumns = {...};
std::vector<std::string> ordersColumns = {...};
std::vector<std::string> customerColumns = {...};
```

These can also be inlined:

```
getRowType("lineitem", {"l_orderkey", "l_quantity"}, lineitemFilePath);
...
```